templates/etc/init.d/elasticsearch.systemd.erb

```diff
@@ -10,9 +10,11 @@ Group=<%= @group %>
 PIDFile=/var/run/elasticsearch/elasticsearch-<%= @name %>.pid
 ExecStart=/usr/share/elasticsearch/bin/elasticsearch -d -p /var/run/elasticsearch/elasticsearch-<%= @name %>.pid -Des.default.config=$CONF_FILE -Des.default.path.home=$ES_HOME -Des.default.path.logs=$LOG_DIR -Des.default.path.data=$DATA_DIR -Des.default.path.work=$WORK_DIR -Des.default.path.conf=$CONF_DIR
 # See MAX_OPEN_FILES in sysconfig
-LimitNOFILE=65535
+LimitNOFILE=<%= @nofile %>
 # See MAX_LOCKED_MEMORY in sysconfig, use "infinity" when MAX_LOCKED_MEMORY=unlimited and using bootstrap.mlockall: true
-#LimitMEMLOCK=infinity
+<% if @memlock == 'unlimited' %>
+LimitMEMLOCK=infinity
+<% end %>
```

Comment:
In case someone doesn't want to set it to infinity perhaps we should add an else statement to use the actual value?
If that would never happen we would need to add some kind of validation of MAX_LOCKED_MEMORY value.